src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -2053,6 +2053,30 @@ public void PreviousSubmissionWithError()
             Assert.Throws<InvalidOperationException>(() => CreateSubmission("a + 1", previous: s0));
         }
 
+        [Fact]
+        public void CreateTupleTypeSymbol()
```

Comment:
Consider the case we are using a `Compilation` that doesn't include a valid definition for `ValueTuple` (for example compiling against vanilla 4.6).  How should this API behave in that case?  I think we should add a test for that.